core/src/main/java/ai/timefold/solver/core/impl/score/director/stream/BavetConstraintStreamScoreDirector.java

```diff
@@ -59,6 +59,7 @@ public BavetConstraintStreamScoreDirector(
     @Override
     public void setWorkingSolution(Solution_ workingSolution) {
         session = scoreDirectorFactory.newSession(workingSolution, constraintMatchPolicy, derived);
+        session.initialize(workingSolution);
```

Comment:
Is this necessary at this stage of the implementation?

Reply:
It was in a previous iteration of this PR, which was doing slightly more work than this version does. Well spotted, I'll remove it.